README.md

```diff
@@ -2,15 +2,15 @@
 Kolibri Instant Schools Plugin
 =================================
 
-How can I install this plugin for development?
+How can I install this plugin for development or local testing?
 ------------------------------
 
-1. Ensure you have (Kolibri)[https://github.com/learningequality/kolibri] setup for local development. Note that all commands below are to be done in the Kolibri repo unless specified otherwise. Also, be sure you're checking out the correct version branch. As of August 2020, this would be `release-v0.12.x`.
+1. Ensure you have (Kolibri)[https://github.com/learningequality/kolibri] setup for local development. **Note that all commands below are to be done in the Kolibri repo unless specified otherwise.** Also, be sure you're checking out the correct version branch. The latest version of Kolibri supported by the plugin is the `release-v0.15.x` branch.
 2. `git clone` this `kolibri_instant_schools_plugin` repository and then `cd` back into where you setup your Kolibri for development.
-3. Set and export the following local variables for use in your development environment: `KOLIBRI_HOME` - this ought to be a new path that was not previously used as a `KOLIBRI_HOME`. Also set `DJANGO_SETTINGS_MODULE` to `kolibri_instant_schools_plugin.instant_schools_settings`.
+3. See the **Env Vars** section below and set the required variables.
 4. Run `pip install -e ../relative/path/to/kolibri_instant_schools_plugin`
-5. Run `kolibri manage migrate --database=instant_schools`
-6. Run `kolibri plugin kolibri_instant_schools_plugin enable` and `kolibri plugin kolibri.plugins.user disable`. If you find that you're having issues, see the next item.
+5. Run `kolibri manage migrate`
+6. Run `kolibri plugin enable kolibri_instant_schools_plugin` and `kolibri plugin disable kolibri.plugins.user_auth kolibri.plugins.default_theme`.
 7. As an alternative to item 6 above, we may instead use plugins files and update our package.json scripts __TEMPORARILY__ in our local Kolibri. Do not commit these changes to Kolibri.
 
 Make your `build_tools/build_plugins.txt` read as follows and remember that this is __TEMPORARY__ and should not be committed to Kolibri:
```

Comment:
I think it'd be easier to leave build_plugins.txt just as:
```
kolibri.core
kolibri.plugins.*
kolibri_instant_schools_plugin
```
I't simpler and worked fine